src/Controller/Api/EventsController.php

```diff
@@ -77,8 +77,8 @@ public function eventsCountInReferentManagedArea(EventRepository $eventRepositor
 
         return new JsonResponse([
             'total' => $eventRepository->countParticipantsInReferentManagedArea($referent),
-            'event_participants' => $eventRegistrationRepository->countEventParticipantsInReferentManagedArea($referent),
-            'in_at_least_one_committee' => $eventRegistrationRepository->countEventParticipantsInReferentManagedAreaInAtLeastOneCommittee($referent),
```

Comment:
Change naming will have no problems with apps using it ?

Reply:
Not used yet ;-)